src/main/java/com/salesforce/mce/awesolog/S3FixedWindowRollingPolicy.java

```diff
@@ -187,4 +197,10 @@ public boolean isRollingOnExit() {
     public void setRollingOnExit(boolean rollingOnExit) {
         this.rollingOnExit = rollingOnExit;
     }
+
+    public boolean isLogbackCredentialPropertiesSet() {
```

Comment:
@steveblackmon maybe also provide option for user to specifically set to use default credential chain?

Reply:
i wound up sticking with just one credential constructor